api-admin/src/test/resources/application.yml

```diff
@@ -18,6 +18,15 @@ env:
   email:
     sendgridApiKey: ${SENDGRID_API_KEY:}
 
+otel:
```

Comment:
for some reason, this is needed now otherwise tests start to instantiate opentelemetry from terra-common-lib